ui/webui/resources/cr_components/certificate_manager/brave_overrides/certificate_manager_v2.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// @ts-nocheck TODO(petemill): Define types and remove ts-nocheck
+
+import {RegisterStyleOverride} from 'chrome://resources/brave/polymer_overriding.js'
+import {html} from 'chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js'
```

Comment:
nit:

```suggestion
import {RegisterStyleOverride} from '//resources/brave/polymer_overriding.js'
import {html} from '//resources/polymer/v3_0/polymer/polymer_bundled.min.js'
```

Reply:
Fixed in [14403c9](https://github.com/brave/brave-core/pull/26561/commits/14403c9f9a9a64e6679ec7a5b66890f3200ee5ac)